README.md

```diff
@@ -246,33 +247,12 @@ python main.py transmit qradar '{"host":"<ip address>", "port":"<port>", "cert":
 python main.py transmit qradar '{"host":"<ip address>", "port":"<port>", "cert":"-----BEGIN CERTIFICATE-----\ncErTificateGoesHere=\n-----END CERTIFICATE-----"}' '{"auth": {"SEC":"1234..sec..uid..5678"}}' is_async
 ```
 
-### Example of converting a STIX pattern to an IBM QRadar AQL query:
-
-[See the QRadar module documentation](stix_shifter/stix_translation/src/modules/qradar/README.md)
-
-### Example of converting IBM QRadar events to STIX:
-
-[See the QRadar module documentation](stix_shifter/stix_translation/src/modules/qradar/README.md)
-
 # Contributing
 
 We are thrilled you are considering contributing! We welcome all contributors.
 
 Please read our [guidelines for contributing](CONTRIBUTING.md).
 
-# Creating new modules
-
-To create a new module that can be used when importing stix-shifter, follow these steps:
-
-- Create a directory with the name of your module in the `stix-shifter/src/modules/` directory
-- In `stix-shifter.py`, add `<module-name>` to the `MODULES` array at the top of the file
-- In your module directory, create a new python code file named `<module-name>_translator.py`. This is where you'll be defining your concrete translator class
-  - In `<module-name>_translator.py`, define a class named `Translator` and have it extend `BaseTranslator` from `stix-shifter/src/modules/base/base_translator.py`. (You can use `stix-shifter/src/modules/dummy/dummy_translator.py` as an example)
-  - In `__init__` you need to assign `self.result_translator` and `self.query_translator` to the appropriate query and result translator you want your module to use. For example the QRadar translator uses `JSONToStix` as its result translator and `StixToAQL` as its query translator
-  - You can write your own query and result translators as well, they must be based off of `BaseQueryTranslator` and `BaseResultTranslator` found in `stix-shifter/src/modules/base/`. Again, you can use the dummy module as a decent example on how to setup the concrete classes found in `stix-shifter/src/modules/dummy/`
-- Once you have this all set up you can invoke your module by running `stix_shifter.py` and passing in your translator module name as the first parameter. The second parameter `query or result` determines if your module runs the query or result translator. The third parameter `data` is passed into your translator as the data that will be translated. If you've imported `stix_shifter.py` into other python code, you can invoke it by running the `translate(module, translation_type, data)` function
```

Comment:
This is not mentioned in the (Developing a new STIX-shifter adapter) guide. Should we add it there?

"Once you have this all set up you can invoke your module by running `stix_shifter.py` and passing in your translator module name as the first parameter. The second parameter `query or result` determines if your module runs the query or result translator. The third parameter `data` is passed into your translator as the data that will be translated. If you've imported `stix_shifter.py` into other python code, you can invoke it by running the `translate(module, translation_type, data)` function"